simba/ui/machine_model_settings_ui.py

```diff
@@ -90,30 +90,78 @@ def __init__(self, config_path: str):
             "25",
             validation="numeric",
         )
-        self.estimators_entrybox.entry_set(val=2000)
+        n_estimators = read_config_entry(
+            self.config,
+            ConfigKey.CREATE_ENSEMBLE_SETTINGS.value,
+            MachineLearningMetaKeys.RF_ESTIMATORS.value,
+            data_type=Dtypes.INT.value,
+        )
+
+        self.estimators_entrybox.entry_set(val=n_estimators)
+        max_features = read_config_entry(
```

Comment:
And same here?         max_features = read_config_entry(self.config, ConfigKey.CREATE_ENSEMBLE_SETTINGS.value, MachineLearningMetaKeys.RF_MAX_FEATURES.value, data_type=Dtypes.STR.value, **default_value=self.max_features_options[0]**)